### DISCUSSION TITLE
Future Language Servers

### DISCUSSION THREAD

**Discussion by ericvicenti** [rise-tools/rise-tools#152] [General]
Right now, Rise Tools only contains a [JS/TS server implementation](https://rise.tools/docs/server-js/). But we are open to supporting more languages, especially if the community is willing to help develop them.

To vote for your favorite server-side language, go to these discussions and upvote your language of choice. If your favorite language is not listed yet, leave a comment or create a thread so we can add it here.

- [PHP](https://github.com/rise-tools/rise-tools/discussions/100)
- [Go](https://github.com/rise-tools/rise-tools/discussions/101)

At the moment of posting, it seems Go has the most votes so far!

**Comment by grabbou** [comment]
I would be also interested in:
- Elixir
- Scala / JVM

Two things come to my mind:
- We should probably look at languages that have similar concept to Rise already, and would be interested in a React (Native)-first solution such as Rise
- Shortlist and interview some companies that take advantage of SDR / SDUI and take notes

**Reply by arpitBhalla** [reply]
https://tech.phonepe.com/introducing-liquidui-phonepes-server-driven-ui-framework/
